portal/server/routes/contract/about/supplierPage.js

```diff
@@ -241,7 +250,6 @@ router.post('/contract/:_id/about/supplier/save-go-back', provide([DEAL, INDUSTR
   }
 
   submissionDetails = industryFields(submissionDetails, industrySectors);
-
```

Comment:
could we please keep line breaks when it improves readability

Reply:
Done